### DISCUSSION TITLE
[ Lidemy HTTP Challenge] 程式導師實驗計畫第四期裡面第五周的作業

### DISCUSSION THREAD

**Discussion by spp72kimo** [Lidemy/forum#265] [其他問題] [answered]
老師好，
想請問您關於這個 Http 的遊戲，我玩到第 7 關的時候，要刪除書籍，卻顯示 Invalid Authorization token，
但我在上一關用的 token 是沒有變過的，有拿到了email，也確定用這個 token 可以 get 查詢書籍資料，但卻沒有辦法 delete 書籍，
想請問老師我有沒有忽略甚麼地方呢？謝謝老師~~

**Comment by aszx87410** [comment, marked as answer]
我剛剛試了一下，應該是沒問題的，你可以把你的 payload 貼上來或是截圖上來我看一下

**Reply by spp72kimo** [reply]
https://imgur.com/a/RNAhBB7
我有把圖片截下來，上面有包含 headers 的資訊，還有 API 的網址，
再麻煩老師幫我看看，謝謝老師！！

**Reply by spp72kimo** [reply]
老師，抱歉！我後來發現問題了，是 needle 這個套件再 delete 的時候 options 前面的參數要加 null，
抱歉，文件沒有看清楚...
已經解決了！謝謝老師~~~
